internal/common/utils/misc.go

```diff
@@ -109,3 +109,16 @@ func LoadSchemas(cfg *viper.Viper, schemaNames []string) (schemas []*jsonschema.
 	}
 	return
 }
+
+func IsOrgIdBlocklisted(cfg *viper.Viper, orgId string) bool {
+	blocklistedOrgIds := strings.Split(cfg.GetString("blocklist.orgids"), ",")
```

Comment:
It is unfortunate that we have to retrieve the block/deny list from the viper object each time and split it into a list.  I wonder if there is a different way to do this.  :thinking: